### ISSUE TITLE
The test results for children are not good

### ISSUE THREAD

**Issue by yanyanlimingkun** [StevenBanama/C3AE#20]
I tested the latest age and gender estimation model on my own private test set with the test script you gave, and found that the effect of age and gender estimation is poor, especially that gender estimation is easy to make mistakes. Have you ever tested it on private test videos? Can you tell me what the problem is?Could it be a problem with tensorflow version

**Comment by StevenBanama** [comment]
> I tested the latest age and gender estimation model on my own private test set with the test script you gave, and found that the effect of age and gender estimation is poor, especially that gender estimation is easy to make mistakes. Have you ever tested it on private test videos? Can you tell me what the problem is?Could it be a problem with tensorflow version

you should add "--white" as a params

**Comment by yanyanlimingkun** [comment]
my command is:
python nets/test.py -g -vid -se -white
I use this command to test, and the effect is still not good

**Comment by yanyanlimingkun** [comment]
I use a video of a child, but the result is in my twenties

**Comment by StevenBanama** [comment]
> my command is:
> python nets/test.py -g -vid -se -white
> I use this command to test, and the effect is still not good. 
can you provide your full command line. I will valid it~.

**Comment by yanyanlimingkun** [comment]
my command is:
python nets/test.py -g -vid -se -white -m ./model/c3ae_model_v2_fp16_white_se_132_4.208622-0.973

**Comment by StevenBanama** [comment]
> I use a video of a child, but the result is in my twenties

if you want detect children, you'd better finetuning in "asia dataset" only. BCZ distribution of utk and afad range from 18~100 and asia dataset range from 0~70.

**Comment by yanyanlimingkun** [comment]
OK, I'll do it. Thank you very much for your help

**Comment by StevenBanama** [comment]
> python nets/test.py -g -vid -se -white -m ./model/c3ae_model_v2_fp16_white_se_132_4.208622-0.973

Can you provide your pic to geekpeakspar@gmail.com. I will test it for you.

I have test many pic of children from baidu, it works normal. you should guarante pic that should be front face and size great than 64 * 64. If you want get a more accurate result, you should build your own dataset.
